artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -170,4 +173,15 @@ private WebAppContext deployWar(String url, String warFile, Path warDirectory) t
       handlers.addHandler(webapp);
       return webapp;
    }
+
+   @Override
+   public void stop() throws Exception {
```

Comment:
@gaohoward if someone called stop() directly, I would expect it to really shutdown.

But I guess it's ok this way.